test/e2e/helm.go

```diff
@@ -0,0 +1,75 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"os/exec"
+
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/cluster-api/test/framework"
+)
+
+// HelmOptions handles arguments to a `helm install` command.
+type HelmOptions struct {
+	StringValues []string // --set-string
+	ValueFiles   []string // --values
+	Values       []string // --set
+}
+
+// InstallHelmChart takes a Helm repo URL, a chart name, and release name, and creates a Helm release on the E2E workload cluster.
+func InstallHelmChart(ctx context.Context, clusterProxy framework.ClusterProxy, namespace, repoURL, chartName, releaseName string, options *HelmOptions, version string) {
```

Comment:
I don't think this `version` parameter is being used at all, but should probably be used to set `--version`.

Reply:
Good catch–I'll update to add `--version` if it's set.